idevice/Cargo.toml

```diff
@@ -65,7 +65,7 @@ springboardservices = []
 misagent = []
 mobile_image_mounter = ["dep:sha2"]
 location_simulation = []
-pair = ["chrono/default", "dep:sha2", "dep:rsa", "dep:x509-cert"]
+pair = ["chrono/default", "dep:sha2", "dep:rsa", "dep:x509-cert", "dep:base64"]
```

Comment:
If you’re modifying a default file (pairing_file.rs), putting the dependency behind a feature gate will cause this to not compile if the pair feature isn’t enabled.